schema/base/land.yaml

```diff
@@ -0,0 +1,74 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Schema
+description: Land
+type: object
+properties:
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Point.json
+      - "$ref": https://geojson.org/schema/LineString.json
+      - "$ref": https://geojson.org/schema/MultiLineString.json
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ./defs.yaml#/$defs/propertyContainers/osmPropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - class
+      - subclass
+    properties:
+      class:
+        description: Broad class of landcover, such as forest, glacier, grass, or a physical feature.
```

Comment:
What is an example of physical feature?

Reply:
Like a mountain peak from OSM, which I've added